_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -120,17 +114,26 @@ $palettes: (
     success: (
```

Comment:
NOTE: needed a few more values added to our system-based colors (for use in demos and grid debugging).